qa/tasks/cephfs/mount.py

```diff
@@ -5,9 +5,9 @@
 import time
 from textwrap import dedent
 import os
-from StringIO import StringIO
 from teuthology.orchestra import run
 from teuthology.orchestra.run import CommandFailedError, ConnectionLostError
+from teuthology.util.compat import stringify
```

Comment:
@kshtsk could you use the helpers from `six` instead? otherwise we have to do the switch all at once: merge all the py3 compat changes in all LTS branches and the py3 compat changes in teuthology in a single shot.

Reply:
We can merge teuthology.util.compat.stringify earlier without a risk, it should not break anything